R/nefsc_groundfish_access.R

```diff
@@ -107,20 +107,21 @@ gmri_survdat_prep <- function(survdat = NULL, survdat_source = "most recent", bo
     "2020"        = paste0(nmfs_path,  "SURVDAT_archived/Survdat_Nye_Aug 2020.RData"),
     "2021"        = paste0(nmfs_path,  "SURVDAT_archived/survdat_slucey_01152021.RData"),
     "bigelow"     = paste0(nmfs_path,  "SURVDAT_current/survdat_Bigelow_slucey_01152021.RData"),
-    "most recent" = paste0(nmfs_path,  "SURVDAT_current/NEFSC_BTS_all_seasons_03032021.RData"),
+    "most recent" = paste0(nmfs_path,  "SURVDAT_current/survdat_lw.rds"),
+    # "most recent" = paste0(nmfs_path,  "SURVDAT_current/NEFSC_BTS_all_seasons_03032021.RData"),
     "bio"         = paste0(nmfs_path,  "SURVDAT_current/NEFSC_BTS_2021_bio_03192021.RData") )
 
 
-
   # If providing a starting point for survdat pass it in:
   if(is.null(survdat) == FALSE){
     survdat <- as.data.frame(survdat)
     trawldat <- janitor::clean_names(survdat)
   } else if(is.null(survdat) == TRUE){
 
     # If not then load using the correct path
-    load(survdat_path)
-    survdat <- as.data.frame(survdat)
+    # load(survdat_path)
+    # survdat <- as.data.frame(survdat)
+    survdat <- readRDS(survdat_path)
```

Comment:
I think we can leave this for now, since we probably only want to work with the most recent going forward, but I don't think the older ones will load with `readRDS` since they are `.RData` extensions. I don't think it is worth adding a checker for filetype here since in my experience I haven't had to toggle survdat versions in a long while.

Reply:
It will break with the "bio" option. But that is easy to circumvent by loading it before using the function.